greenmail-core/src/main/java/com/icegreen/greenmail/smtp/commands/StartTlsCommand.java

```diff
@@ -0,0 +1,24 @@
+package com.icegreen.greenmail.smtp.commands;
+
+import java.io.IOException;
+
+import com.icegreen.greenmail.smtp.SmtpConnection;
+//import com.icegreen.greenmail.smtp.SmtpHandler;
+import com.icegreen.greenmail.smtp.SmtpManager;
+import com.icegreen.greenmail.smtp.SmtpState;
+
+public class StartTlsCommand extends SmtpCommand {
+
+	@Override
+	public void execute(SmtpConnection conn, SmtpState state, SmtpManager manager, String commandLine)
+			throws IOException {
+        //Check for any parameters/extra illegal characters sent
+        if (!commandLine.toUpperCase().equals("STARTTLS")) {
```

Comment:
Hmm I probably should have added a whitespace strip in here to be a little kinder.